pkg/integration/components/runner.go

```diff
@@ -138,12 +139,15 @@ func prepareTestDir(
 	return createFixture(test, paths, rootDir)
 }
 
-func buildLazygit(raceDetector bool) error {
+func buildLazygit(debug bool, raceDetector bool) error {
 	// // TODO: remove this line!
 	// // skipping this because I'm not making changes to the app code atm.
 	// return nil
 
 	args := []string{"go", "build"}
+	if debug {
+		args = append(args, "-gcflags=all=-N -l")
```

Comment:
Can we add an inline comment explaining what these flags mean?

Reply:
Sure, see 955fc4f36ce2.